src/logic/__tests__/projectSuccessfullySavedTest 2.js

```diff
@@ -0,0 +1,17 @@
+import projectSuccessfullySaved from '../projectSuccessfullySaved';
```

Comment:
Seems like we got duplicates of this and some other files? Note the ` 2` in the filename…